test/fixtures/linter/sqlfluffignore/path_c

```diff
@@ -1 +1 @@
-.
\ No newline at end of file
+.
```

Comment:
@kayman-mk this seems to be the change that's upsetting windows, investigating further

Reply:
lol why is there an infinitely repeating symlink here
![image](https://user-images.githubusercontent.com/80432516/146615768-da6f7c52-7385-4d33-8385-d2beb1cf1826.png)

Reply:
Interestingly when I ran `tox -e pre-commit -- run --all-files` I had all the same files get fixed except this path_c symlink:
![image](https://user-images.githubusercontent.com/80432516/146616973-94273886-5f17-4a79-827b-803131f4b1cd.png)

Reply:
@kayman-mk out of curiosity which OS are you using and did you run the same command as me?

I think to be safe I'll add this file to the exludes in pre-commit-config

Reply:
No idea what's going on here. Using Windows with IntelliJ, but no Python installed. So I do the pre-commit in a Alpine docker image.